scripts/cm_boot_public_ip_linux.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+HOST_NAME=$(curl ifconfig.me)
```

Comment:
I can't find any information on who owns this site, uptime guarantees etc. It seems safer to use something like akamai? http://whatismyip.akamai.com/ Also a bit more self explanatory if an http is included in front.

Also, I think it'd be better to rename this file to `cm_boot_linux_public_ip.sh` and rename `cm_boot_linux.sh` to `cm_boot_linux_localhost.sh` to clarify the difference and so that all linux scripts are grouped together.